k8s/namespaces/fees-pay/ccpay-bulkscanning-api/ccpay-bulkscanning-api.yaml

```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: fees-pay
```

Comment:
Can you remove this from here please, set from kustomize level

Reply:
have removed it from fees-register and bulk-scan as well